tardis/plasma/properties/nlte_rate_equation_solver.py

```diff
@@ -27,6 +27,8 @@ def calculate(
         rate_matrix_index,
         number_density,
         nlte_excitation_species,
+        coll_exc_coeff,
+        coll_deexc_coeff,
```

Comment:
the number of arguments here is temporary, a lot of them will get moved once NLTE excitation is done